src/api-usability.bib

```diff
@@ -0,0 +1,61 @@
+@article{myers2016improving,
+  title={Improving API usability},
+  author={Myers, Brad A and Stylos, Jeffrey},
+  journal={Communications of the ACM},
+  year={2016},
+}
+
+
+@article{scheller2015automated,
+  title={Automated measurement of API usability: The API concepts framework},
```

Comment:
LaTeX will change the capitalization of things like "API", so you should put braces around that throughout the .bib file: `{API}`